### DISCUSSION TITLE
Turbo ACC 网络加速

### DISCUSSION THREAD

**Discussion by baodp** [immortalwrt/immortalwrt#895] [Q&A]
### Describe the bug

请问怎么在X86 的编译出的固件中有Turbo ACC 网络加速？没找到呀。

### ImmortalWrt version

r25436-5da18713f3

### ImmortalWrt target/subtarget

x86/64

### Device

INTEL HM570

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by baodp** [comment]
还有怎么在web管理的主页显示CPU温度 网卡温度？

**Comment by baodp** [comment]
https://github.com/immortalwrt/immortalwrt/discussions/830

原来是被去除了。。。

**Comment by laaArmy** [comment]
这玩意有用是有用，但是会导致一堆BUG

**Reply by ghost** [reply]
这对硬路由应该是有用的，尝试添加第三方turbo ACC编译失败。

**Reply by cherinyy** [reply]
> 这对硬路由应该是有用的，尝试添加第三方turbo ACC编译失败。

如果是联发科的话，不需要，即便是 OpenWrt 主线都已经支持硬件卸载。
如果是高通，我不清楚 SFE 是否有效，手上没有设备。
